src/contract.js

```diff
@@ -64,20 +68,29 @@ export class Contract {
   }
 
   /**
-   * Returns the read-only footprint entry necessary for any invocations to this
-   * contract, for convenience when adding it to your transaction's overall
-   * footprint.
+   * Returns the read-only footprint entries necessary for any invocations to
+   * this contract, for convenience when adding it to your transaction's overall
+   * footprint or doing bump/restore operations.
    *
-   * @returns {xdr.LedgerKey} the contract's executable data ledger key
+   * @returns {xdr.LedgerKey[]} the ledger keys containing the contract's code
```

Comment:
the typedef for [Contract.getFootPrint()](https://github.com/stellar/js-stellar-base/blob/better-contract-footprint/types/index.d.ts#L33) should reflect this different return type ?

Reply:
ugh yes thank you so much